src/positioner/src/getPosition.js

```diff
@@ -328,16 +341,24 @@ function getPosition({
   }
 
   const topRectFitsOnTop = getFitsOnTop(topRect, viewportOffset)
+
   const bottomRectFitsOnBottom = getFitsOnBottom(
     bottomRect,
     viewport,
     viewportOffset
   )
 
-  if (positionIsAlignedOnTop && topRectFitsOnTop) {
-    return {
-      position,
-      rect: topRect
+  if (positionIsAlignedOnTop) {
```

Comment:
This behaviour makes more sense to me - that it would alter the position to fit below if the popover fits within the viewport... but perhaps I'm missing something.

Reply:
I am not exactly sure how this is different in behavior from what's currently there. An example screenshot would help!

Reply:
@jeroenransijn Here are some gifs that demonstrate the difference. With the old logic, the popover will be moved down even if the entire popover could instead be positioned below the button. With the new logic, the popover will check if it can appear entirely below the button. Only if it can't, will it reposition it within its preferred position. 

Old:
![evergreen-position-old](https://user-images.githubusercontent.com/1236841/45108681-238fc900-b0f2-11e8-88c6-85796d3a288e.gif)

New:
![evergreen-position-new](https://user-images.githubusercontent.com/1236841/45108721-402c0100-b0f2-11e8-8e19-5f7f92cb0a83.gif)

Reply:
Looks better!